src/utils/components/DiskModal/utils/helpers.ts

```diff
@@ -136,7 +136,11 @@ export const getDataVolumeFromState = ({
     resultVolume?.persistentVolumeClaim?.claimName ||
     `${vm?.metadata?.name}-${diskState.diskName}`;
 
-  dataVolume.metadata.name = dvName;
+  const isTemplateParameter = /[${}]/.test(dvName);
```

Comment:
Please consider exporting to a function as the same code is repeated three times.
nameWithoutParameter(name, diskStateDiskName, defaultValue?)
abusively any other way of function can work as u choose.

Reply:
done thanks for the suggestion